Adição da main.cpp

andrebarbieri opened the pull request.

andrebarbieri committed:
Create main.cpp

doug-nicihoka committed:
Leves mudanças e correções main.cpp

Adição de # no include <iostream>, melhor indentação, adição de using namespace std.

doug-nicihoka commented on EP2/main.cpp:
```diff
@@ -0,0 +1,93 @@
+    cout << "Obter quantos pontos? ";
+    cin >> quantidade;
+    Serie* s = new Serie(nomeDaSerie, canalX, canalY);
+
```
Falta a impressão de _"Obtendo os pontos"_, ao longo da obtenção dos pontos da **Serie**. 


`cout << "Obtendo os pontos" << endl;` 
`    for (int i = 0; i < quantidade; i++) {`
____ `        is->atualizar();`
____ `        serie->adicionar(is->getValor(canalX), is->getValor(canalY));`
`    }`

doug-nicihoka committed:
Mudança nome de variáveis e else ifs

Melhor indentação,  váriável "tipoX" e "tipoY' alteradas para "tipoEixoX" e "tipoEixoY", respectivamente. Os elses foram mudados para else ifs.

doug-nicihoka reviewed (changes requested):
Os objetos Eixo criados nas linhas 58, 66, 80 e 88 são de escopo local, ou seja, existem apenas dentro dos "ifs" e "else ifs". Declarar os objetos de maneira global (fora das condicionais).

doug-nicihoka commented on EP2/main.cpp:
```diff
@@ -0,0 +1,93 @@
+        cin >> maximoPadraoEixoY;
+        EixoDinamico *eixoY = new EixoDinamico(minimoPadraoEixoY, maximoPadraoEixoY, s, false);
+    }
+    Grafico* g = new Grafico(eixoX, eixoY, s);
```
Gera erro de compilação.

> error: 'eixoX' was not declared in this scope

> error: 'eixoY' was not declared in this scope

doug-nicihoka committed:
(IMPORTANTE!) Grandes mudanças na main

Objetos "eixoX" e "eixoY" definidos globalmente (linhas 54 e 79). 

Mudança de local das variáveis "tipoEixoX" e "tipoEixoY" (linhas 53 e 78).

Alteração do nome da serie (de "s" para "serie"). 

Obtenção de pontos agora implementada (linhas 46 a 50).

doug-nicihoka committed:
Correção de impressões

Quando o usuário digita 'e', deve-se imprimir "Valor maximo/minimo: ".
Já quando o usuário digita 'd', deve-se imprimir "Valor maximo/minimo padrao: "

doug-nicihoka committed:
Remoção de acentuação

Acentuação no compilador causa problemas.
